csm/conf/setup.py

```diff
@@ -247,14 +279,21 @@ def store_encrypted_password(conf_data):
                 raise InvalidPillarDataError(f"failed to get pillar data for {const.OPENLDAP}")
             sspl_config = SaltWrappers.get_salt_call(const.PILLAR_GET, const.SSPL)
             if sspl_config and type(sspl_config) is dict:
+                Log.info("SSPL Credentials Copied to CSM Configuration.")
                 conf_data[const.CHANNEL][const.USERNAME] = sspl_config.get(const.USERNAME)
                 conf_data[const.CHANNEL][const.PASSWORD] = sspl_config.get(const.PASSWORD)
             else:
                 raise InvalidPillarDataError(f"failed to get pillar data for {const.SSPL}")
+            _paswd = Setup._fetch_csm_user_password()
+            if not _paswd:
+                raise CsmSetupError("CSM Password Not Set by Provisioner.")
+            Log.info("CSM Credentials Copied to CSM Configuration.")
+            conf_data[const.CSM][const.PASSWORD] = _paswd
             cluster_id = SaltWrappers.get_salt_call(const.GRAINS_GET, const.CLUSTER_ID)
             provisioner_data = conf_data[const.PROVISIONER]
             provisioner_data[const.CLUSTER_ID] = cluster_id
             conf_data[const.PROVISIONER] = provisioner_data
+            # NKP get csm user/password using salt command and set to conf 
```

Comment:
@navalkp Please check whether this comment is required. If not remove it.

Reply:
will remove.